safe/gui/tools/batch/batch_dialog.py

```diff
@@ -681,6 +694,7 @@ def generate_pdf_report(self, impact_function, iface, scenario_name):
         dialog. This function is adapted from analysis_utilities.py
         :param impact_function: Impact Function
```

Comment:
Can you update it to use code standard?
param variable_name: explanation.
type variable_name: type of the variable.